docs/Migration-Guide-V3.md

```diff
@@ -89,6 +89,17 @@ const schema = {
 fastify.route({ method, url, schema, handler });
 ```
 
+If you use `fastify-swagger` to generate API documentation, you need to use `fastify.getSchema` instead of `$ref`.
```

Comment:
This is true **now**, it will not be true when fastify will exit from `rc`: that plugin needs only to be updated.

I would add this sentence only if we are unable to do that task in the near future